skimage/feature/blob.py

```diff
@@ -166,7 +166,7 @@ def _prune_blobs(blobs_array, overlap):
 
 
 def blob_dog(image, min_sigma=1, max_sigma=50, sigma_ratio=1.6, threshold=2.0,
-             overlap=.5,):
+             overlap=.5, exclude_border=False):
```

Comment:
```suggestion
             overlap=.5, *, exclude_border=False):
```

We are using [keyword-only arguments](https://www.python.org/dev/peps/pep-3102/) going forward as a way to get more flexibility when modifying the API in the future.